apps/main-app/src/components/pathway/pathwayWidget.js

```diff
@@ -465,44 +465,46 @@ class PathwayWidget extends Component {
     return (
       <HorizontalScroll className='text-nowrap'>
         <div id="modPathway" style={gocstyles}>
-
-            {(this.state.gocams.loaded && this.state.gocams.list && this.state.gocams.list.length > 0) ?
-                <div style={{ "padding": "1rem 0.2rem" }}>
-                <span style={{ "paddingRight": "1rem"}}>Available GO-CAMs: </span>
+          {
+            (this.state.gocams.loaded && this.state.gocams.list && this.state.gocams.list.length > 0) ?
+              <div style={{ "padding": "1rem 0.2rem" }}>
+                <span  pan style={{ "paddingRight": "1rem"}}>Available GO-CAMs: </span>
```

Comment:
Is this a typo?

Reply:
Yeah, I think it is .. thanks for noticing!  Fix coming soon.